### DISCUSSION TITLE
Is there any authentication method?

### DISCUSSION THREAD

**Discussion by jhomarolo-vortx** [Kong/kong-manager#162] [Ideas]
Is there any type of authentication for this interface? I believe that leaving it open could be a security flaw.

It would be nice to have some type of authentication, even if it was basic (username and password).

**Comment by nasrulhazim** [comment]
For me, at the moment I put it behind reverse proxy with basic authentication configured. 

But, another issue is to the Admin API is not secured. I already [requested](https://github.com/Kong/kong-manager/discussions/154) if it's possible to have the Admin API secure as well.
